src/components/app/App.tsx

```diff
@@ -70,7 +70,7 @@ const App: React.FC = () => {
         isDevelopment={isDevelopment()}
         isTest={isTest()}
         schemaString={getSchemaString()}
-        credentials="include"
+        credentials={isDevelopment() || isTest() ? "" : "include"}
```

Comment:
This is dependent on whether we require credentials for our local gql server

Reply:
we do require them